pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/QuickstartRunner.java

```diff
@@ -93,9 +95,31 @@ public QuickstartRunner(List<QuickstartTableRequest> tableRequests, int numContr
     _enableTenantIsolation = enableIsolation;
     _authToken = authToken;
     _configOverrides = configOverrides;
+    _zkAddress = null;
+    _deleteExistingData = true;
```

Comment:
this should be added as a command-line parameter for QuickStartCommand?

Reply:
I've refactored the constructor to make this simpler. For deleting existing data, if the flag is set to `false` for any of the other quickstart commands, there will be an exception on startup when attempting to create a data directory that already exists. I've added `_deleteExistingData` as a constructor argument for future quickstart commands.